05_syntax_lab/01.py

```diff
@@ -1,5 +1,13 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+"""
+Print thr bigest number between 10 inputs
+"""
+newX=0

```

Comment:
Hi Gil And Welcome,

This works great when some inputs are positive, but if all inputs are negative you'll get the wrong result.

Can you find the absolute lowest value in python?